frontend/app/src/store/reports/const.ts

```diff
@@ -2,11 +2,27 @@ import {
   ProfitLossOverviewData,
   ReportError,
   ReportPeriod
-} from '@/store/reports/types';
+} from '@rotki/common/lib/reports';
 import { Zero } from '@/utils/bignumbers';
 
-export const MUTATION_PROGRESS = 'progress' as const;
-export const MUTATION_REPORT_ERROR = 'reportError' as const;
+export enum ReportActions {
+  FETCH_REPORTS = 'fetchReports',
+  FETCH_REPORT = 'fetchReport',
+  EDIT_REPORT = 'editReport'
+}
+
+export enum ReportMutations {
+  SET_REPORT = 'set',
+  SET_REPORTS = 'setReports',
+  ADD_REPORT = 'addReport',
+  CURRENCY = 'currency',
+  REPORT_PERIOD = 'reportPeriod',
+  ACCOUNT_SETTINGS = 'accountSettings',
+  RESET = 'reset'
+}
+
+export const MUTATION_PROGRESS = 'progress';
+export const MUTATION_REPORT_ERROR = 'reportError';
```

Comment:
Since you already introduced the enum above, would it be possible to also make these two part of the enum?